### ISSUE TITLE
Keyboard focus does not move to the newly open window after invoke Sign In Form.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#107]
User Experience: 

After pressing enter key on any control, if the visual focus does not moves to the newly open window then the users who depends upon keyboard/Screen Reader for navigation might get affected and hence it has to use extra TAB to reach the next interactive element available on the newly open window.

 

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode

4. Navigate to the Left nav and select Pages button.

5. Navigate to the Pages window. Select "APIs: List" option.

6. Navigate to the any editable control details and select Add widget.

7. Navigate to the Sign In Form and select it.

8. Navigate to the various controls of the "Sign In Form" dialog box.

9. Verify, whether the visual focus indicator is visible on First interactive Element of newly open window or not.

 

Actual Result:

After pressing enter key on add Sign in form, then the visual focus does not move to the first interactive element of the newly open window.

 

Expected Result:

After pressing enter key on add Sign in form, then the visual focus should moves to the first interactive element of the newly open window, so that the user can able to navigate properly on the controls available on the open window.

 

MAS Reference:

MAS 3.2.1 – On Focus (06)

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

**Comment by azaslonov** [comment]
Fixed.
